components/MonitoringUserSetupInstructions.tsx

```diff
@@ -103,8 +116,8 @@ $$ LANGUAGE plpgsql VOLATILE SECURITY DEFINER;`}
         data: it ensures that the collector cannot piggyback other queries that could exfiltrate data.
       </p>
       <p>
-        Because EXPLAIN needs to run in the same database where the query ran, you will need to create this function
-        <strong>in each database you want to monitor</strong>. You should create the function as a Postgres superuser to ensure EXPLAIN
+        Because EXPLAIN needs to run in the same database where the query ran, you will need to create this function <strong>in each
```

Comment:
Because of JSX whitespace rules, this rendered as "...function**in each database**...", so just fixing that.